<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QWidget" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>小姐姐喊你收消息啦V1.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="com.qrc">
    <normaloff>:/新前缀/icon.jpg</normaloff>:/新前缀/icon.jpg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { 

	color: rgb(2, 15, 255);
    font-size:20px;
	background-color: white;
}

QTextBrowser#textBrowser1{
	font-family:微软雅黑; 
    color: white ;
    font-size:20px;
padding:10px 10px;
}

QLabel#label{ 
	font-family:迷你简黄草;
    color: white ;
    font-size:22px;
}
QLabel#label1{ 
	font-family:迷你简黄草;
    color: white ;
    font-size:20px;
}
QLabel#label2{ 
	font-family:迷你简黄草;
    color: white ;
    font-size:20px;
}
QLabel#label_2{ 
	font-family:	Cooper Black;
    color: white ;
    font-size:20px;
}

.QWidget{ background-color: black }

QComboBox{
	font: 75 9pt &quot;Arial&quot;;
    color: black ;
    font-size:20px;
}

QPlainTextEdit{
	padding:5px 5px;
	font-family:微软雅黑;
	    font-size:20px;
}

QPushButton#button3{
margin-top : 2px;
margin-bottom:0px
padding:10px 10px;
}

QPushButton#button4{
margin-top : 2px;
margin-bottom:0px
padding:10px 10px;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="pixmap">
          <pixmap resource="com.qrc">:/新前缀/logo.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="label1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>迷你简黄草</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>选择端口</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox1">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>-1</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertAtCurrent</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label2">
             <property name="font">
              <font>
               <family>迷你简黄草</family>
               <pointsize>-1</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>波特率</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>6</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>-1</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertAtCurrent</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToMinimumContentsLength</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="duplicatesEnabled">
              <bool>false</bool>
             </property>
             <property name="modelColumn">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>21</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="button4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>迷你简黄草</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>搜索串口</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button1">
         <property name="font">
          <font>
           <family>迷你简黄草</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button5">
         <property name="font">
          <font>
           <family>迷你繁陆行</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>清空缓存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Cooper Black</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>BY-ZNJ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="8,2">
     <item>
      <widget class="QTextBrowser" name="textBrowser1">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/新前缀/girl.jpg);


</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPlainTextEdit" name="TextEdit">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>59</number>
         </property>
         <item>
          <widget class="QPushButton" name="button3">
           <property name="font">
            <font>
             <family>迷你简黄草</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button2">
           <property name="font">
            <font>
             <family>迷你简黄草</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
 </widget>
 <resources>
  <include location="com.qrc"/>
 </resources>
 <connections/>
</ui>
